Doc/glossary.rst

```diff
@@ -424,11 +424,11 @@ Glossary
       An object that tries to find the :term:`loader` for a module that is
       being imported.
 
-      Since Python 3.3, there are two types of finder: :term:`meta path finders
+      Since Python 2.3, there are two types of finder: :term:`meta path finders
```

Comment:
```suggestion
      There are two types of finder: :term:`meta path finders
```
We generally don't keep historical notes for Python 2.

Reply:
Thank you @encukou  for the suggested change that we generally do not keep historical notes for Python 2. Given this, the revision you proposed aligns with our documentation standards. However, I would like to clarify if there might be any exceptions where historical context is crucial for understanding current functionality, or if we should strictly adhere to this rule across all documentation. Your guidance on this would be valuable.